### PR TITLE
budget: update entities budget and acq_account for new formly component

### PR DESCRIPTION
**Pull request by Garfield-fr**
* Updates budget and acq_account for formly.
* Fixes translation for new options form.
* Updates babel config to extract new label key.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Why are you opening this PR?

- update entities budget and acq_account.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: rero_ils/modules/budgets/jsonschemas/budgets/budget-v0.0.1.json [hunk 1]
```diff
@@ -24,26 +33,60 @@
...
+          }
+        }
+      }
     },
     "end_date": {
       "type": "string",
       "format": "date",
+      "title": "Budget end date",
       "pattern": "\\d{4}-((0[1-9])|(1[0-2]))-(((0[1-9])|[1-2][0-9])|(3[0-1]))$",
       "validationMessage": "Required. Should be in the ISO 8601, YYYY-MM-DD.",
```
Comment (iGormilhit) [inline rero_ils/modules/budgets/jsonschemas/budgets/budget-v0.0.1.json]: Is this line still relevant?

File: rero_ils/modules/budgets/jsonschemas/budgets/budget-v0.0.1.json [hunk 2]
```diff
@@ -24,26 +33,60 @@
...
-      "title": "Budget start date"
+      "form": {
+        "type": "datepicker",
+        "wrappers": [
+          "form-field"
+        ],
+        "validation": {
+          "messages": {
+            "pattern": "Should be in the ISO 8601, YYYY-MM-DD."
```
[the remark this answers is not in the document]
Comment (Garfield-fr) [reply rero_ils/modules/budgets/jsonschemas/budgets/budget-v0.0.1.json]: done

File: rero_ils/modules/budgets/jsonschemas/budgets/budget-v0.0.1.json [hunk 3]
```diff
@@ -24,26 +33,60 @@
...
       "validationMessage": "Required. Should be in the ISO 8601, YYYY-MM-DD.",
-      "title": "Budget end date"
+      "form": {
+        "type": "datepicker",
+        "wrappers": [
+          "form-field"
+        ],
+        "validation": {
+          "messages": {
+            "pattern": "Should be in the ISO 8601, YYYY-MM-DD."
```
[the remark this answers is not in the document]
Comment (Garfield-fr) [reply rero_ils/modules/budgets/jsonschemas/budgets/budget-v0.0.1.json]: done

File: rero_ils/modules/acq_accounts/jsonschemas/acq_accounts/acq_account-v0.0.1.json [hunk 4]
```diff
@@ -28,7 +33,10 @@
       "title": "Name",
       "description": "Name of the Acquisition account.",
```
[the remark this answers is not in the document]
Comment (Garfield-fr) [reply rero_ils/modules/acq_accounts/jsonschemas/acq_accounts/acq_account-v0.0.1.json]: done

File: rero_ils/modules/budgets/jsonschemas/budgets/budget-v0.0.1.json [hunk 5]
```diff
@@ -24,26 +33,60 @@
...
+          }
+        }
+      }
     },
     "start_date": {
       "type": "string",
       "format": "date",
+      "title": "Budget start date",
       "pattern": "\\d{4}-((0[1-9])|(1[0-2]))-(((0[1-9])|[1-2][0-9])|(3[0-1]))$",
       "validationMessage": "Required. Should be in the ISO 8601, YYYY-MM-DD.",
```
[the remark this answers is not in the document]
Comment (Garfield-fr) [reply rero_ils/modules/budgets/jsonschemas/budgets/budget-v0.0.1.json]: I make only `Start date` on title

File: rero_ils/modules/budgets/jsonschemas/budgets/budget-v0.0.1.json (hunk 1 above)
Comment (Garfield-fr) [reply rero_ils/modules/budgets/jsonschemas/budgets/budget-v0.0.1.json]: I make only `End date` on title

**Pull request merged**